docs/apm/real-user-monitoring/configure-data-collection.md

```diff
@@ -47,7 +64,7 @@ To configure a RUM HTTP Traces source:
    ```
    * **Probabilistic sampling rate** (optional): Add a **Probabilistic sampling rate** for heavy traffic sites in a decimal value based on percentage, for example, 10% would be entered as `0.1`.
    * **Ignore urls** (optional): Add a list of URLs not to collect trace data from. Supports regex. For example: `/^https:\/\/www.tracker.com\/.*/, /^https:\/\/api.mydomain.com\/log\/.*/`. Please make sure provided URLs are valid JavaScript flavor regexes.
-   * **Custom Tags** (optional): Click **+Add** and enter a key and value for each **Custom Tags** to show in spans from instrumented browsers. For example, click **+Add** and enter a key `deployment.environment` with a value of `production`. This information is saved in the script for `name_of_your_web_service`.
+   * **Custom Tags** (optional): Click **+Add** and enter a key and value for each **Custom Tags** to show in spans from instrumented browsers. *replace removed example with another one like internal.version=0.1.21* This information is saved in the script for `name_of_your_web_service`.
```

Comment:
not sure what's going on here. I asked to provide new example not meant to place the text directly.

Reply:
Please my [reply on our thread about this](https://github.com/SumoLogic/sumologic-documentation/pull/1414#discussion_r1087400182) asking for a specific example.

Reply:
Can you or @mat-rumian please provide a specific key/value I can use here as an example?

Reply:
you can use `internal.version=0.1.21` as an example